tests/test_rpc.py

```diff
@@ -1157,3 +1158,85 @@ def test_getserverstatus(revault_network, bitcoind):
         res = w.rpc.call("getserverstatus")
         for watchtower in res["watchtowers"]:
             assert not watchtower["reachable"]
+
+
+@pytest.mark.skipif(not POSTGRES_IS_SETUP, reason="Needs Postgres for servers db")
+def test_gethistory(revault_network, bitcoind, executor):
+    """
+    Retrieve the event History and check the presence of some triggered events.
+    """
+
+    CSV = 12
+    revault_network.deploy(2, 2, n_stkmanagers=1, csv=CSV)
+    mans = revault_network.mans()
+    stks = revault_network.stks()
+
+    t1 = int(time.time())
+
+    # Create one canceled vault.
+    canceled_vault = revault_network.fund(0.05)
+    revault_network.secure_vault(canceled_vault)
+    revault_network.activate_vault(canceled_vault)
+    revault_network.spend_vaults_anyhow_unconfirmed([canceled_vault])
+    revault_network.cancel_vault(canceled_vault)
+
+    # We wait a little before and after the timestamp to keep everything
+    # cleanly seperated
+    time.sleep(1)
+    t2 = int(time.time())
+    time.sleep(1)
```

Comment:
I really dislike `sleep()`s in tests as they are very often a cause of races which makes tests flaky. Here they make more sense given we play with timestamps, but i think they aren't actually needed.
At least that's what i can tell from 3 iterations of:
```
while true; do cargo build --release && BITCOIND_PATH=$PWD/bitcoind-22.0 TIMEOUT=60 REVAULTD_PATH=$PWD/target/release/revaultd POSTGRES_USER=revault POSTGRES_PASS=revault pytest --ignore tests/servers/ -n 20 --dist each -k test_gethistory; done
```

Reply:
You are right, I think I wrote it when timestamps were not from block time and were relative to each participant synchronization. I must have updated it my bad